cmd/fyne_settings/settings/bundled.go

```diff
@@ -16,7 +16,9 @@ var appearanceIcon = &fyne.StaticResource{
+*/
```

Comment:
See the first line in this file - this change will get reverted the next time the file is generated. Instead remove https://github.com/fyne-io/fyne/blob/master/cmd/fyne_settings/data/gen.go#L60

Reply:
Yeah good catch. I don't know why I didn't think of that :sweat_smile: